apparmor.d/groups/virt/cni

```diff
@@ -0,0 +1,35 @@
+abi <abi/3.0>,
+
+include <tunables/global>
+
+profile loopback /{opt/,}{cni/,}bin/loopback {
```

Comment:
```suggestion
@{exec_path} = /opt/cni/bin/loopback
profile loopback @{exec_path} {
```

Same for the other profiles.

Reply:
I've switched to using the exec_path, but I'm not sure if using a hardcoded path is the correct way. I use k3s with Calico, but kubeadm with Cilicium may do completely different things.